language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -808,4 +829,35 @@ protected void sortByAssignability(BallerinaCompletionContext context, LSComplet
 
         completionItem.getCompletionItem().setSortText(sortText);
     }
+
+    /**
+     * Get the completion item list for client declaration module prefix reference, when the node is within the 
+     * qualified name reference node.
+     * 
+     * @param context   Ballerina Completion Context
+     * @param qNameRef  Qualified Name Reference Node
+     * @param predicate Predicate to filter symbols
+     * @return  Client Declaration Completion Item List
+     */
+    protected List<LSCompletionItem> getClientDeclCompletionItemList(BallerinaCompletionContext context, 
+                                                                     QualifiedNameReferenceNode qNameRef, 
+                                                                     Predicate<Symbol> predicate) {
+        Optional<ClientDeclSymbol> clientDeclSymbol = ModuleUtil.searchClientDeclarationForAlias(context,
+                QNameRefCompletionUtil.getAlias(qNameRef));
+        List<Symbol> clientDeclContent = clientDeclSymbol.map(symbol -> symbol.moduleSymbol().allSymbols().stream()
+                        .filter(predicate)
+                        .collect(Collectors.toList()))
+                .orElseGet(ArrayList::new);
+
+        List<LSCompletionItem> completionItemList = this.getCompletionItemList(clientDeclContent, context);
+        for (LSCompletionItem item: completionItemList) {
+            CompletionItem cItem = item.getCompletionItem();
+            if (cItem.getLabel().equals("'client")) {
+                cItem.setLabel("client");
+                cItem.setInsertText("client");
```

Comment:
Shall we use `ItemResolverConstants.CLIENT` instead?

Reply:
Fixed in 84d8264e9b050ba2c3f6823e8d4282ea576c8a7a